angular/src/components/set-password.component.ts

```diff
@@ -18,19 +18,25 @@ import { EncString } from 'jslib-common/models/domain/encString';
 import { SymmetricCryptoKey } from 'jslib-common/models/domain/symmetricCryptoKey';
 
 import { KeysRequest } from 'jslib-common/models/request/keysRequest';
+import { OrganizationUserResetPasswordEnrollmentRequest } from 'jslib-common/models/request/organizationUserResetPasswordEnrollmentRequest';
 import { SetPasswordRequest } from 'jslib-common/models/request/setPasswordRequest';
 
 import { ChangePasswordComponent as BaseChangePasswordComponent } from './change-password.component';
 
 import { HashPurpose } from 'jslib-common/enums/hashPurpose';
 import { KdfType } from 'jslib-common/enums/kdfType';
+import { PolicyType } from 'jslib-common/enums/policyType';
+
+import { Utils } from 'jslib-common/misc/utils';
 
 @Directive()
 export class SetPasswordComponent extends BaseChangePasswordComponent {
     syncLoading: boolean = true;
     showPassword: boolean = false;
     hint: string = '';
     identifier: string = null;
+    orgId: string;
+    resetPasswordAutoEnroll = false;
```

Comment:
Instead of having a class level variable for this set onInit we should wrap that init logic into a function that returns orgId if true, and just call when needed before making API calls.

Reply:
Hmm. Could go either way with this one. I'm not sure how I feel about the ambiguity of `orgId != null` instead of an explicit `boolean`. The boolean provided supplies the `ngIf` conditional for all downstream UI implementations (at this moment). Will think on this!

Reply:
Ah, okay, I was missing the context that these values were used in the UI (yay jslib). It makes sense to leave this as is.